### ISSUE TITLE
Won't run on Node.js v7

### ISSUE THREAD

**Issue by lukechilds** [asapach/peerflix-server#88]
Had everything running fine on Node.js v5.10.1, upgraded to v7.0.0 and got the following error:

```
$ peerflix-server
/root/.nvm/versions/node/v7.0.0/lib/node_modules/peerflix-server/node_modules/socket.io/lib/store.js:35
Store.prototype.__proto__ = EventEmitter.prototype;
                                        ^

TypeError: Cannot read property 'prototype' of undefined
    at Object.<anonymous> (/root/.nvm/versions/node/v7.0.0/lib/node_modules/peerflix-server/node_modules/socket.io/lib/store.js:35:41)
    at Module._compile (module.js:573:32)
    at Object.Module._extensions..js (module.js:582:10)
    at Module.load (module.js:490:32)
    at tryModuleLoad (module.js:449:12)
    at Function.Module._load (module.js:441:3)
    at Module.require (module.js:500:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/root/.nvm/versions/node/v7.0.0/lib/node_modules/peerflix-server/node_modules/socket.io/lib/manager.js:16:13)
    at Module._compile (module.js:573:32)
``` 

Reverting back to v5.10.1 resolved the issue.

**Comment by tanmng** [comment]
I recently updated nodejs to version ```7.1.0``` and have the same issue

**Comment by asapach** [comment]
OK, it looks like it's time to update `socket.io`

**Comment by asapach** [comment]
Fixed in [`v0.1.0`](https://github.com/asapach/peerflix-server/releases/tag/v0.1.0)
